substrate/primitives/npos-elections/fuzzer/Cargo.toml

```diff
@@ -17,7 +17,7 @@ workspace = true
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
-clap = { version = "4.5.1", features = ["derive"] }
+clap = { version = "4.5.2", features = ["derive"] }
```

Comment:
We should now be able to pull up all external deps to the workspace level. Testing in https://github.com/polkadot-fellows/runtimes/pull/201 was sucessful.